src/index.js

```diff
@@ -1,5 +1,6 @@
-import Provider, { createProvider } from './components/Provider'
+import Provider from './components/Provider'
 import connectAdvanced from './components/connectAdvanced'
+import Context from './components/Context'
 import connect from './connect/connect'
 
-export { Provider, createProvider, connectAdvanced, connect }
+export { Provider, connectAdvanced, Context, connect }
```

Comment:
If we're going to export it, let's call it `ReactReduxContext` or something like that.

Reply:
Under UMD, this is going to be `ReactRedux.Context`. In ES land, you can rename it yourself if it collides (`import { Context as ReduxContext }`)